extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/statistics/QuantilesSketchKeyCollectorFactory.java

```diff
@@ -38,9 +38,9 @@
 public class QuantilesSketchKeyCollectorFactory
     implements KeyCollectorFactory<QuantilesSketchKeyCollector, QuantilesSketchKeyCollectorSnapshot>
 {
-  // smallest value with normalized rank error < 0.1%; retain up to ~86k elements
+  // smallest value with normalized rank error < 0.01%; retain up to ~430k elements
   @VisibleForTesting
-  static final int SKETCH_INITIAL_K = 1 << 12;
+  static final int SKETCH_INITIAL_K = 1 << 15;
```

Comment:
So as this change would have some implications on how much memory we use, can we document this as part of the PR description?